src/main/kotlin/no/nav/familie/ba/soknad/api/personopplysning/PersonopplysningerService.kt

```diff
@@ -1,27 +1,53 @@
 package no.nav.familie.ba.soknad.api.personopplysning
 
+import no.nav.familie.kontrakter.felles.personinfo.Bostedsadresse
```

Comment:
Hvis dere tar Option+Cmd+L for å formatte denne hadde det vært nais, men ikke et must.